appveyor.yml

```diff
@@ -108,7 +108,7 @@ build_script:
   - copy /Y Far-NetBox\*.md ci\%configuration%.%platform%\Plugins\NetBox
   - copy /Y Far-NetBox\LICENSE.txt ci\%configuration%.%platform%\Plugins\NetBox
   #build FarColorer
-  - set COLORER_VERSION=1.4.4
+  - for /F "delims=xxx" %a in ('curl -fsSLJ --ssl-no-revoke "https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt"') do set COLORER_VERSION=%a
```

Comment:
- `%` should be doubled in scripts, e.g. `%%a`  in both places (probably that's why it fails now).
- `"delims=xxx"` should probably be `"delims="`.

Reply:
why do you need --ssl-no-revoke? would be better without it.

Reply:
it wasn`t work without --ssl-no-revoke on local test in windows console